public/icons/Spinner2.svg

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" style="margin: auto; background: none; display: block; shape-rendering: auto;" width="24px" height="24px" viewBox="0 0 100 100" preserveAspectRatio="xMidYMid">
+<g transform="rotate(0 50 50)">
+  <rect x="47" y="24" rx="3" ry="4.32" width="6" height="12" fill="#ffffff">
```

Comment:
If you use same colour multiple times in svg(for example `fill="#ffffff"`) it's better to put it in variable and just import it everywhere. More than that, in case you'll decide to change the whole app theme, it would be much easier to change the colors.

Reply:
Fixed!